src/scss/components/_login-button-logo.scss

```diff
@@ -1,17 +1,37 @@
+@use '~microscope-sass/lib/bem';
+
 @import '~microscope-sass/lib/grid';
 @import '~microscope-sass/lib/responsive';
 
 @import '../mixins/prefix';
 
 .#{prefix(login-button-logo)} {
-  display: flex;
+  display: inline-flex;
```

Comment:
using `inline-flex` removes the need to limit the height/width to an 'arbitrary' pixel value.